crates/forge/tests/e2e/fork_warning.rs

```diff
@@ -66,7 +66,7 @@ fn should_print_warning() {
             r"
                 [..]Compiling[..]
                 [..]Finished[..]
-                [WARNING] The RPC node with url = http://188.34.188.184:9545/rpc/v0_5 has unsupported version = (0.5.1), use node supporting RPC version ({EXPECTED_RPC_VERSION})
+                [WARNING] RPC node with the url http://188.34.188.184:9545/rpc/v0_5 uses incompatible version 0.5.1. Expected version: {EXPECTED_RPC_VERSION}
```

Comment:
[nitpick]: I believe this is forge's convention to use `=` extensively in their messages, so we might want to leave those here. If I remember correctly, as a part of consistency between forge and cast, they also tried to incorporate those to our messages some time ago.

Reply:
After taking a brief look, it doesn't seem to be very consistent throughout forge.

@piotmag769 can you confirm whether you'd prefer using `=` in this warning? Personally I don't like it, but if this is a convention I can change it.

Reply:
I'm fine with whatever, if noone from forge objects, this can be as is.